Makefile

```diff
@@ -0,0 +1,43 @@
+KUEUE_REPO=https://github.com/kubernetes-sigs/kueue.git
+KUEUE_TMP_PATH=/tmp/xpk_tmp/kueue
+
+KUBECTL_VERSION := $(shell curl -L -s https://dl.k8s.io/release/stable.txt)
+
+KUBECTL_AMD64="https://dl.k8s.io/release/$(KUBECTL_VERSION)/bin/linux/amd64/kubectl"
+KUBECTL_ARM="https://dl.k8s.io/release/$(KUBECTL_VERSION)/bin/linux/arm64/kubectl"
```

Comment:
Maybe create a single variable with a conditional based on the OS?

Reply:
This will add new lines, I think I will leave it like here